### PR TITLE
TodoList-Feature

### PR DESCRIPTION
**Pull request by rgora23**
- Updated spacing for code in the TodoList.tsx
- Add TodoListItem and TodoListApp styled components
- Add X button to delete todo list item

### CODE REVIEW

File: src/pages/TodoList.tsx [hunk 1]
```diff
@@ -1,40 +1,57 @@
...
+          setInputValue(e.target.value)
+        }} />
+
+      <Button onClick={handleAddTodoItem}>Add Item</Button>
+
+      <ul>
+        {todoList.map((value, i) => {
+          return (
+            <TodoListItem>
+              {/* The onclick prop takes an anonymous function that filters out the item clicked from the todoList state */}
```
Comment (lim-is-a) [inline src/pages/TodoList.tsx:45]: Please remove the commented out lines.

File: src/pages/TodoList.tsx [hunk 2]
```diff
@@ -1,40 +1,57 @@
...
+const TodoListItem = styled.div`
+  display: flex;
+  align-items: center;
+  width: 40%;
+  justify-content: left;
+`
+
+const TodoListApp = styled.div`
+  text-align: center;
+`
```
Comment (lim-is-a) [inline src/pages/TodoList.tsx:16]: Why did you add these styled components?

**Review by lim-is-a — CHANGES_REQUESTED** [review]

File: src/pages/TodoList.tsx (hunk 1 above)
Comment (rgora23) [reply src/pages/TodoList.tsx:45]: whoops forgot to remove this!

File: src/pages/TodoList.tsx (hunk 2 above)
Comment (rgora23) [reply src/pages/TodoList.tsx:16]: Its easier to maintain the styling of those components. They were simple divs before this and all the styles needed to be passed in manually. 

This way we only need to change these components at the top and it'll reflect in our todolist app

**Review by lim-is-a — APPROVED** [review]
